src/webgpu/api/validation/resource_usages/texture/in_render_common.spec.ts

```diff
@@ -115,6 +115,11 @@ g.test('subresources,color_attachments')
       .combine('inSamePass', [true, false])
       .unless(t => t.inSamePass && t.level0 !== t.level1)
   )
+  .beforeAllSubcases(t => {
+    if (t.isCompatibility) {
```

Comment:
This feels like the `if (t.isCompatibility)` check should be moved to `selectDeviceForRendrableColorFormatOrSkipTestCase`. It's on the test, it knows if it's in compat mode or not.

Reply:
Yes this is redundant. removed